x-pack/plugins/security_solution/server/lib/detection_engine/signals/executors/threshold.ts

```diff
@@ -70,11 +70,21 @@ export const thresholdExecutor = async ({
   primaryTimestamp: string;
   secondaryTimestamp?: string;
   aggregatableTimestampField: string;
+  filter: Filter | undefined;
+  unprocessedExceptions: ExceptionListItemSchema[];
 }): Promise<SearchAfterAndBulkCreateReturnType & { state: ThresholdAlertState }> => {
   const result = createSearchAfterReturnType();
   const ruleParams = completeRule.ruleParams;
 
   return withSecuritySpan('thresholdExecutor', async () => {
+    if (unprocessedExceptions.length !== 0) {
+      ruleExecutionLogger.warn(
+        `The following exceptions won't be applied to rule execution: ${JSON.stringify(
+          unprocessedExceptions
+        )}`
+      );
+    }
```

Comment:
Let's extract this to a function we can call from the 3 rule types that need the warning, and also format the unprocessedExceptions in a more presentable way than stringifying the whole object